tools/package-tools/compile-entry-point.ts

```diff
@@ -44,6 +57,16 @@ export function renamePrivateReExportsToBeUnique(buildPackage: BuildPackage,
   }
 }
 
+/** Adds `importAs` property to generated all metadata.json files for an entry-point. */
+function addImportAs(packageName: string, outputPath: string, secondaryEntryPoint: string): void {
+  const path = join(outputPath, secondaryEntryPoint);
+  glob(join(path, '**/*.+(metadata.json)')).forEach(metadataPath => {
+    let metadata = JSON.parse(readFileSync(metadataPath, 'utf-8'));
```

Comment:
nit: Rather than `JSON.parse(readFileSync(...))`, you should be able to do `require(...)`, although that'll put the file in npm's cache.